controller/setting_controller.go

```diff
@@ -600,50 +589,73 @@ func (sc *SettingController) updateTaintToleration() error {
 	if err != nil {
 		return errors.Wrap(err, "failed to list backing image manager pods for toleration update")
 	}
+	pods := append(imPodList, smPodList...)
+	pods = append(pods, bimPodList...)
+	for _, pod := range pods {
+		updatingRuntimeObjects = append(updatingRuntimeObjects, runtime.Object(pod))
+	}
```

Comment:
Can we move the updatingRuntimeObjects collection codes to a helper function, for example
```
updatingRuntimeObjects, err := collectRuntimeObjects(...)
```

Same for other functions.

Reply:
>Can we move the updatingRuntimeObjects collection codes to a helper function

Do you mean that move similar codes to `utils` or `go-common-libs`?

Reply:
Modified. Thanks